app/controllers/predicates/AuthPredicate.scala

```diff
@@ -65,20 +65,20 @@ class AuthPredicate @Inject()(authService: EnrolmentsAuthService,
           if (enrolments.enrolments.exists(_.key == AuthKeys.agentEnrolmentId)){
             authoriseAsAgent(block)
           } else {
-            logger.debug("[AuthPredicate][invokeBlock] - Agent does not have correct agent enrolment ID")
+            debug("[AuthPredicate][invokeBlock] - Agent does not have correct agent enrolment ID")
             Future.successful(Forbidden(unauthorisedAgent()))
           }
 
         case Some(_) ~ enrolments => authoriseAsNonAgent(enrolments, block)
         case None ~ _ =>
-          logger.warn("[AuthPredicate][invokeBlock] - Missing affinity group")
```

Comment:
same here

Reply:
i think this should be kept as errorLog since it kicks out the user from the journey if they dont have an affinity group